src/scripts/modules/add_destination.js

```diff
@@ -28,15 +28,11 @@ function cacheDom() {
 
 function bindEvents() {
   form.addEventListener("submit", (event) => {
-    addDestination();
+    addDestination(formState);
     event.preventDefault();
-    //get the index of the destination pushed to the destinationsList and publish it
-    const itemIndex = destinationsList.length - 1;
-    events.publish("destinationAdded", itemIndex);
-    // storeNew();
+    storeNew();
```

Comment:
This code `storeNew()` only handles logic. Do you think is it the right place to be here or... or should it be... `store.js`? That's right, it should be moved to keep the store logic together. That code can be moved to inside `function addDestination`. Having it there, will help you understand how to toggle the visit item